test/jdk/java/foreign/TestUpcall.java

```diff
@@ -28,7 +28,7 @@
  * @modules jdk.incubator.foreign/jdk.internal.foreign
  * @build NativeTestHelper CallGeneratorHelper TestUpcall
  *
- * @run testng/othervm
+ * @run testng/othervm/timeout=720
```

Comment:
I've increased the timeout of TestUpcall here as well, as I was seeing timeouts pretty frequently